Security

```diff
@@ -0,0 +1,93 @@
+Security
+OpenMRS is in contact with Security PS, an enterprise security consulting company.Security PS donated time to enhance the security of OpenMRS. Their application security experts found and privately reported software security weaknesses to the OpenMRS team. 
+The development team have analyzed Security PS’s results and have made and are making plans to address the discovered issues. 
+The org.openmrs.util.Security class provides basic encryption and decryption methods for use in the API.
+The security class used by Andro client is :
+java.lang.Object
+org.openmrs.util.Security and then public class that has to be implemented or added extends Object class.
```

Comment:
No need to go into so much depth, remember, this is a guide is for any user who wants to use Android Client and he/she may not have knowledge about coding.

Reply:
But until and unless a user is not aware about the class used under which those encryption and decryption methods are being used for password security how could he use those features

Reply:
You can just write that "we use _some algorithm_ which use _xxx bit key_ and fairly secure". You can also add 1-2 sentence about the working of the algorithm.

Reply:
what about these methods? should i remove all of them?